lib/verdict/storage/base_storage.rb

```diff
@@ -43,7 +45,7 @@ def store_start_timestamp(experiment, timestamp)
         set(experiment.handle.to_s, 'started_at', timestamp.utc.strftime('%FT%TZ'))
       end
 
-
```

Comment:
the convention in verdict has been to use `protected` instead of `private`, to reveal the intention that classes are expected to inherit from this one, and to possibly implement those methods themselves.